src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -752,7 +755,7 @@ def set_up_from_revision(self):
         if self.get_argument_from_revision():
             r = None
             try:
-                r = self.client.show_revision(cmd=self.cmd, resource_group_name=self.get_argument_resource_group_name(), container_app_name=self.get_argument_name(), name=self.get_argument_from_revision())
+                r = self.client.show_revision(cmd=self.cmd, resource_group_name=self.get_argument_resource_group_name(), container_app_name=self.get_argument_name(), name=self.get_argument_from_revision(), show_sensitive_values=self.get_argument_show_sensitive_values())
```

Comment:
`set_up_from_revision()` is a function, which get template for existed revision, as requestBody, should not be impacted by  argument show_sensitive_values.